internal/oauth2/endpoints/endpoints.go

```diff
@@ -0,0 +1,25 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package endpoints
+
+import (
+	"golang.org/x/oauth2"
+)
+
+var Dynatrace = oauth2.Endpoint{
```

Comment:
nitpick: I don't think there is any need for this to be an `oauth2.Endpoint`? 
If I'm not mistaken we only ever pull the TokenURL string out when loading and use it as string everywhere - so I think we could simply remove the unnecessary dependency on oauth2 and define a string in here.

Reply:
fine to keep, as it's standard practice in other libs/packages :+1: